Spectra v2.7.2 (#3365)
brainstormforce/ultimate-addons-for-gutenberg: README.md

## Changelog ##

### x.x.x ###

> * Fix: Display Conditions - Resolved an issue wherein the `UAGDay` attribute would persist in the code view after turning off display conditions.
> * Fix: Social Share - Converted anchors to span tags and removed unwanted attributes from the social links.
> * Fix: Forms - Resolved an issue in which some phone country-code colors would not work as intended in the editor.
> * Fix: Table Of Content - Resolved an issue where default padding of Table of Content is not same in editor as in frontend.

Reviewer: Reorder required, with some minor improvements.

```suggestion
* Fix: Forms - Resolved an issue in which some phone country-code colors would not work as intended in the editor.
* Fix: Table Of Contents - Resolved an issue wherein the default padding not same in the editor and on the frontend.
* Fix: Social Share - Converted anchor tags to span tags and removed unwanted attributes from the social links.
* Fix: Display Conditions - Resolved an issue wherein the `UAGDay` attribute would persist in the code view after turning off display conditions.
```

Reviewer 2: Done.
